test/ulid/rails_test.rb

```diff
@@ -39,6 +39,9 @@ def test_manual_id_generation
   end
 
   def test_validate_ulid_format
+    # Validation was introduced in 5.0, https://github.com/rails/rails/commit/8e633e505880755e7e366ccec2210bbe2b5436e7
+    return if "4.2" == "#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}"
```

Comment:
The validation method was introduced in rails 5.0 but there are ways to do validation with rails 4.2.

I am personally not a fan of different behaviour between different dependency versions 🤷🏼‍♀️

Reply:
I'd be happy to work with you to provide consistent validation between the supported Rails versions.